02-KernelModules_CharacterDevice/Create_device.h

```diff
@@ -0,0 +1,94 @@
+#pragma once
```

Comment:
Header files in the Linux kernel are used for two purposes:
    1. to define interfaces between components of the kernel, and
    2. to define interfaces between the kernel and user space 
Plz put function implementations into .c files.